pkg/controller/account/iam.go

```diff
@@ -139,59 +134,6 @@ func deleteAllAccessKeys(client awsclient.Client, iamUser *iam.User) error {
 	return nil
 }
 
-// checkIAMUserExists checks if a given IAM user exists within an account
-// Takes a logger, an AWS client for the target account, and a target IAM username
-func checkIAMUserExists(reqLogger logr.Logger, client awsclient.Client, userName string) (bool, *iam.GetUserOutput, error) {
```

Comment:
Was this removed because its not called anywhere?

Reply:
See https://github.com/openshift/aws-account-operator/pull/472#discussion_r488721276

Reply:
+1, since we moved that function to pkg/awsclient/iam.go